server_admin/topics/user-federation/ldap.adoc

```diff
@@ -195,3 +195,38 @@ must be enabled explicitly in the LDAP configuration page. See the documentation
 
 WARNING: Always verify that user passwords are properly hashed and not stored as plaintext by inspecting a changed
 directory entry using `ldapsearch` and base64 decode the `userPassword` attribute value.
+
+[[_ldap_troubleshooting]]
+==== Troubleshooting
+
+- It is useful to increase the logging level to TRACE for the category `org.keycloak.storage.ldap` . This can be done in the logging
+subsystem in the `standalone(-ha).xml` file. With this setting, there is big amount
+of logging messages sent to the `server.log` file in the `TRACE` level, including the logging for all queries to the LDAP server and the parameters, which were
+used to send the queries. When you are creating any LDAP question on user forum or JIRA, it can be useful to attach the server log with
+enabled TRACE logging. If it is too big, the good alternative is to include just the snippet from server log with the messages, which were
+added to the log during the operation, which causes the issues to you.
+
+
+- When you create LDAP provider, there is message displayed in the server log in the INFO level starting with:
+```
+Creating new LDAP Store for the LDAP storage provider: ...
+```
+It shows the configuration of your LDAP provider. Before you are asking the questions or reporting bugs, it will be nice to include this
+message to show your LDAP configuration. Eventually feel free to replace some config changes, which you do not want to include, with some
+placeholder values. Like for example `bindDn=some-placeholder` . For `connectionUrl`, feel free to replace it as well, but it is generally
```

Comment:
Change
"Like for example `bindDn=some-placeholder` ."
to
One example is `bindDn=some-placeholder`."